source/viz.md

```diff
@@ -1055,14 +1091,63 @@ glue('can_lang_plot_theme', can_lang_plot_theme, display=False)
 Scatter plot of percentage of Canadians reporting a language as their mother tongue vs the primary language at home colored by language category with color-blind friendly colors.
 :::
 
-From the visualization in {numref}`can_lang_plot_theme`,
+The chart above gives a good indication of how the different language categories differ,
+and this information is sufficient to answer our research question.
+But what if we want to know exactly which language correspond to which point in the chart?
+With a regular visualization library this would not be possible,
+as adding text labels for each individual language
+would add a lot of visual noise and make the chart difficult to interpret.
+However, since Altair is an interactive visualization library we can add information on demand
+via the `Tooltip` encoding channel,
```

Comment:
this is cool, but we need to be very careful with how we include this due to the print version of the book obviously not being able to show interactive figures. 

Is it possible to render a different image in the PDF version of the book that is just a static screenshot of what the tooltip would look like? If not immediately, please open an issue to sort that out (and anywhere else we add interactive figures where it's important that the interactivity is displayed -- though I think it will only matter here in Ch4)

Reply:
Good point, tracking in https://github.com/UBC-DSCI/introduction-to-datascience-python/issues/175